Sources/Nimble/Matchers/Predicate.swift

```diff
@@ -183,7 +183,7 @@ extension Predicate {
 
 // Backwards compatibility until Old Matcher API removal
 @available(*, deprecated, message: "Use Predicate directly instead")
-extension Predicate: Matcher {
+extension Predicate {
```

Comment:
This is a breaking change, so please wait for Nimble v10 for a while.

Reply:
I don't agree, because this change will make it clearer which matcher to use, the old deprecated or the new one. And it won't break any code :)
Example:
```
func to<U>(_ matcher: U, description: String? = nil) -> Self where U: Matcher, U.ValueType == T
func to(_ predicate: Predicate<T>, description: String? = nil) -> Self
```

Reply:
I can't say that removing the protocol conformance doesn't break downstream Nimble users' usages at all (libraries which depend on Nimble's public APIs).